packages/hop-node/src/cli/swap.ts

```diff
@@ -8,23 +8,30 @@ import { BigNumber, utils as ethersUtils } from 'ethers'
 import { Chain, TokenIndex } from 'src/constants'
 import { actionHandler, logger, parseBool, parseNumber, parseString, root } from './shared'
 
-import { swap as uniswapSwap } from 'src/uniswap'
+import { swap as oneInchSwap } from 'src/1inch'
 
 root
   .command('swap')
   .description('Swap tokens on Uniswap or via Hop AMM')
   .option('--chain <slug>', 'Chain', parseString)
   .option('--from <symbol>', 'From token', parseString)
+  .option('--from-native [boolean]', 'From native token', parseBool)
   .option('--to <symbol>', 'To token', parseString)
-  .option('--amount <number>', 'From token amount', parseNumber)
+  .option('--to-native [boolean]', 'To native token', parseBool)
```

Comment:
IICU, thie `--to-native` and the above `--from-native` options are now artifacts and can be removed.

Reply:
good catch, removed in https://github.com/hop-protocol/hop/commit/a38b614cc480ac4cfdd492433b6bb30041374d73